docs/src/developer-guide/working-with-rules.md

```diff
@@ -179,11 +182,17 @@ This method returns the scope which has the following types:
 | `CatchClause`             | `catch`    |
 | others                    | ※3        |
 
-**※1** Only if the configured parser provided the block-scope feature. The default parser provides the block-scope feature if `parserOptions.ecmaVersion` is not less than `6`.<br>
-**※2** Only if the `for` statement defines the iteration variable as a block-scoped variable (E.g., `for (let i = 0;;) {}`).<br>
-**※3** The scope of the closest ancestor node which has own scope. If the closest ancestor node has multiple scopes then it chooses the innermost scope (E.g., the `Program` node has a `global` scope and a `module` scope if `Program#sourceType` is `"module"`. The innermost scope is the `module` scope.).
-
-The returned value is a [`Scope` object](scope-manager-interface) defined by the `eslint-scope` package. The `Variable` objects of global variables have some additional properties.
+**※1** Only if the configured parser provided the block-scope feature.
+The default parser provides the block-scope feature if `parserOptions.ecmaVersion`
+is not less than `6`.<br>
+**※2** Only if the `for` statement defines the iteration variable as
+a block-scoped variable (E.g., `for (let i = 0;;) {}`).<br>
+**※3** The scope of the closest ancestor node which has own scope.
+If the closest ancestor node has multiple scopes then it chooses the innermost scope
+(E.g., the `Program` node has a `global` scope and a `module` scope
+if `Program#sourceType` is `"module"`. The innermost scope is the `module` scope.).
+
+The `Variable` objects of global variables have the following additional properties:
```

Comment:
where is this in the `Scope` object? the [`Scope.variables: Variable[]` property](https://eslint.org/docs/latest/developer-guide/scope-manager-interface#variables)?

Reply:
Yup, exactly. The definition of the `Scope` class is here in source control, for reference:
https://github.com/eslint/eslint-scope/blob/4e1d24ca4a747c14b37f059543cf08d1e1820b2d/lib/scope.js#L156